front-end/src/app/reports/submission-workflow/submit-report-status.component.html

```diff
@@ -1,133 +1,100 @@
-            {{ activeReportSignal().formLabel }}
-          </h3>
-          <br />
-        </div>
-        @if (reportCodeSignal()) {
-          <div class="col-12">
-            <h5>YOUR REPORT TYPE:</h5>
-            <h3>
-              {{ reportCodeLabelMap ? reportCodeLabelMap[reportCodeSignal()] : '' }}
-            </h3>
-            <br />
-          </div>
-        }
-        @if (coverageDatesSignal()) {
-          <div class="col-12">
-            <h5>YOUR COVERAGE DATES:</h5>
-            <h3 class="no-wrap">
-              {{ coverageDatesSignal()!['coverage_from_date'] || null | longDate }} —
-              <wbr />
-              {{ coverageDatesSignal()!['coverage_through_date'] || null | longDate }}
-            </h3>
-            <br />
-          </div>
-        }
-
-        <div class="col-12">
-          <h5 class="word-break">CONFIRMATION EMAILS:</h5>
-          <h3 class="word-break">
-            {{ activeReportSignal().confirmation_email_1 }} <br />
-            {{ activeReportSignal().confirmation_email_2 }}
-          </h3>
-          <br />
-        </div>
-        @if (activeReportSignal().upload_submission?.fec_report_id) {
-          <div class="col-12">
-            <h5>FILING ID:</h5>
-            <h3>
-              {{ activeReportSignal().upload_submission?.fec_report_id }}
-            </h3>
-          </div>
+<div class="mt-5 ml-2 p-component">
+  <h1>Report status</h1>
+  <span class="subtitle">
+    <strong>{{ activeReportSignal().formLabel }}: </strong>{{ activeReportSignal().formSubLabel }}
+  </span>
+  <hr type="solid" />
+  <div class="flex">
+    <div class="col-4 pl-0">
+      @if (reportStatusSignal() === reportStatus.SUBMIT_FAILURE) {
+        <h2 class="submission-status-heading">Report submission failed</h2>
+        <p class="submission-status-text">Please contact the Electronic Filing Office or visit FEC.gov for help.</p>
+        @if (fecStatusSignal() === 'REJECTED' && fecMessageSignal()) {
+          <p class="error-text">Error Message: {{ fecMessageSignal() }}</p>
         }
-      </div>
-    </div>
-    <div class="col-8">
-      @switch (fecStatusSignal()) {
-        @case ('ACCEPTED') {
-          <img
-            class="image-max-width"
-            ngSrc="assets/img/Report-status--Congratulations.png"
-            draggable="false"
-            alt="Form submitted success image"
-            height="271"
-            width="450"
-          />
+        <h3 class="assistance-heading">Need help?</h3>
+        <a
+          pButton
+          href="https://www.fec.gov/help-candidates-and-committees/filing-reports/upload-errors/"
+          target="_blank"
+          rel="noopener"
+          class="p-button-secondary learn-more-button"
+        >
+          Learn more about your submission error
+        </a>
+        <h3 class="assistance-heading">Electronic Filing Office</h3>
+        <p class="assistance-text">1-800-424-9530, menu option 4 202-694-1307 efiletechsupport&#64;fec.gov</p>
```

Comment:
ac had a typo,  the mockup has this saying "menu option 4, or 202..." rather than "menu option 4 202..."

Reply:
new update.  we're making it match https://www.fec.gov/contact/
Shannon is updating the wireframes.
So the text stays as you've got it and we just need to add some newlines in between